<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>25</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#exitButton{border-image: url(:/img_normal/exit.png);background:transparent;
}

QPushButton#exitButton:hover{border-image: url(:/img_hover/exit_hover.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimumButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>25</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#minimumButton{
border-image: url(:/img_normal/hide.png);
background:transparent;}
QPushButton#minimumButton:hover{
border-image: url(:/img_hover/hide_hover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bglabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img_background/mainbackground.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="haedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>25</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(200,200,200,150);
border-radius:3px;
background:transparent;
color:white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>28</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img_ico/search.png);
background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>900</width>
      <height>575</height>
     </rect>
    </property>
    <widget class="QListWidget" name="modeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>575</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:rgb(248,248,248)</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>54</width>
       <height>54</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>airlines</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="Resources/buttons.qrc">
        <normaloff>:/img_ico/airlines.ico</normaloff>:/img_ico/airlines.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>booked</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/buttons.qrc">
        <normaloff>:/img_ico/booked.ico</normaloff>:/img_ico/booked.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>manger</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/buttons.qrc">
        <normaloff>:/img_ico/manager.ico</normaloff>:/img_ico/manager.ico</iconset>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>820</width>
       <height>575</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="addNewAirlineButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>475</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#addNewAirlineButton{border-image: url(:/img_normal/add.png);}

QPushButton#addNewAirlineButton:hover{border-image: url(:/img_normal/add_hover.png);}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="userInfoLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>15</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white</string>
    </property>
    <property name="text">
     <string>&lt;p&gt;韩金伟&lt;/p&gt;</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="arthorLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>649</y>
      <width>900</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2ca9e1</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>899</x>
      <y>75</y>
      <width>2</width>
      <height>575</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2ca9e1</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>75</y>
      <width>2</width>
      <height>575</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2ca9e1</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="bookTicketWidget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>820</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:15px;
border-top-right-radius:15px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(44, 129, 225, 65), stop:1 rgba(44, 129, 255, 10));
</string>
    </property>
    <widget class="QLabel" name="bookTicketInfolabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>方正姚体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bookTicketAcceptbtn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#bookTicketAcceptbtn{border-radius:6px;
background-color:/*#44cef6;*/
#2ca9e1;
color:#ffffff}
QPushButton#bookTicketAcceptbtn:hover{background-color:#44cef6;}
QPushButton#bookTicketAcceptbtn:pressed{background-color:#007bbb}</string>
     </property>
     <property name="text">
      <string>立即购买</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bookTicketrejectBtn">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#bookTicketrejectBtn{border-radius:6px;
background-color:/*#44cef6;*/
rgb(248,248,248);

color:black}
QPushButton#bookTicketrejectBtn:hover{background-color:#e9e7ef;}
QPushButton#bookTicketrejectBtn:pressed{background-color:#c2ccd0}</string>
     </property>
     <property name="text">
      <string>再想想</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="headButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="warningLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>15</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:transparent
</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'等线 Light'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:10pt; color:#ffffff;&quot;&gt;亲爱的管理员请注意！&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:10pt; color:#ffffff;&quot;&gt;你的所有改动都会在数据库生效！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="shadowWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{background-color:rgba(0,0,0,0.6);}</string>
    </property>
   </widget>
   <zorder>bglabel</zorder>
   <zorder>haedLabel</zorder>
   <zorder>lineEdit</zorder>
   <zorder>searchButton</zorder>
   <zorder>infoWidget</zorder>
   <zorder>userInfoLabel</zorder>
   <zorder>arthorLabel</zorder>
   <zorder>bookTicketWidget</zorder>
   <zorder>line</zorder>
   <zorder>line_3</zorder>
   <zorder>line_2</zorder>
   <zorder>headButton</zorder>
   <zorder>warningLabel</zorder>
   <zorder>minimumButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>shadowWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/buttons.qrc"/>
 </resources>
 <connections/>
</ui>
